File: sourceCodez/Python/palindrome.py
#palindrome number

'''

a number which is the same even if it is reversed is known as palindrome

101
121
132231
12421
'''

n=int(input('enter a number :'))
a=n
s=0
while(a!=0):
    r=a%10
    s=s*10+r
    a=a//10
if(s==n):
    print('palindrome')
else:
    print('not palindrome')

File: specificIDEprojectz/blueJ/Numbers/Palindrome.java
/*
A number is a Palindrome which when read in reverse order is same as 
read in the right order
Example : 11, 101, 151 etc.
                                                        [ICSE 2008]
*/
public class Palindrome
{
public static void main(int n)
{
int b,d,r=0;
b=n;
while(b!=0)
{
d=b%10;
r=r*10+d;
b=b/10;
}
if(r==n)
System.out.println(n+" is a Palindrome number");
else
System.out.println(n+" is not a Palindrome number");
}
}